sdk/cognitiveservices/azure-ai-inkrecognizer/azure/ai/inkrecognizer/_client.py

```diff
@@ -0,0 +1,324 @@
+)
+
+from ._enums import InkStrokeKind, ApplicationKind, InkPointUnit, ServiceVersion
+from ._models import _parse_recognition_units
+
+
+_LOGGER = logging.getLogger(__name__)
+
+_DEFAULT_ARGUMENTS = {
+    "application_kind": ApplicationKind.MIXED,
+    "ink_point_unit": InkPointUnit.MM,
+    "language": "en-US",
+    "unit_multiple": 1.0,
+    "service_version": ServiceVersion.PREVIEW
+}
+
+
+def _validate_param(param, param_name, types):
+    if not isinstance(types, (list, tuple)):
+        types = [types]
+    if not isinstance(param, tuple(types)):
+        if len(types) == 1:
+            supported_type = "<%s>" % types[0].__name__
+        else:
+            supported_type = "in (%s)" % (", ".join(t.__name__ for t in types))
+        if param is None:
+            type_got = "None"
+        else:
+            type_got = type(param).__name__
+        error_info = "%s should be %s, got <%s>." % (param_name, supported_type, type_got)
+        raise ValueError(error_info)
+
+
+def _get_and_validate_param(kwargs, param_name, types, default=None):
+    value = kwargs.pop(param_name, None)
+    if value is None:
+        return default
+
+    _validate_param(value, param_name, types)
+    return value
+
+
+class _AzureConfiguration(Configuration):
+    def __init__(self, credential, **kwargs):  # pylint:disable=super-init-not-called
+        self._set_universal(**kwargs)
+        # sync-specific azure pipeline policies
+        scopes = []
+        if "scopes" in kwargs:
+            scopes = kwargs.pop("scopes")
+        self.authentication_policy = BearerTokenCredentialPolicy(credential, *scopes, **kwargs)
+        self.retry_policy = kwargs.get("retry_policy", RetryPolicy(**kwargs))
+        self.redirect_policy = kwargs.get("redirect_policy", RedirectPolicy(**kwargs))
+        self.transport = kwargs.get("transport", RequestsTransport())
+
+    def _set_universal(self, **kwargs):
+        Configuration.__init__(self)
+        self.connection = kwargs.get("connection_configuration", ConnectionConfiguration(**kwargs))
+        self.user_agent_policy = kwargs.get("user_agent_policy", UserAgentPolicy(**kwargs))
+        self.proxy_policy = kwargs.get("proxy_policy", ProxyPolicy(**kwargs))
+        self.logging_policy = kwargs.get("logging_policy", NetworkTraceLoggingPolicy(**kwargs))
+        self.headers_policy = kwargs.get("headers_policy", HeadersPolicy(**kwargs))
+
+
+class _InkRecognizerConfiguration():
+    def __init__(self, **kwargs):
+        # Ink Recognizer request arguments
+        self.application_kind = _get_and_validate_param(kwargs, "application_kind", ApplicationKind)
+        self.ink_point_unit = _get_and_validate_param(kwargs, "ink_point_unit", InkPointUnit)
+        self.language = _get_and_validate_param(kwargs, "language", str)
+        self.unit_multiple = _get_and_validate_param(kwargs, "unit_multiple", (int, float))
+        self.service_version = _get_and_validate_param(kwargs, "service_version", ServiceVersion)
```

Comment:
ServiceVersion should allow a string value.

Reply:
See reply of the other comment